src/network/task.rs

```diff
@@ -360,9 +390,11 @@ impl NetworkTask {
     }
 
     pub async fn run(mut self) -> Result<()> {
-        let mut io = NetworkIO::new(self.iface, self.net_tx.clone(), self.py_tx.clone());
+        let mut io = NetworkIO::new(self.iface, self.net_tx.clone());
         let mut remove_conns = Vec::new();
 
+        let mut send_py_permit: Option<Permit<TransportEvent>> = None;
```

Comment:
Nit-pick: why `send_py_permit` and not `py_tx_permit`? The different naming scheme confused me at first.

Reply:
Yeah, fair point. Let's change it. 👍